src/DankCord/gateway.py

```diff
@@ -79,10 +79,13 @@ def __boot_ws(self):
         identify_json = orjson.loads(identify)
 
         for guild in identify_json["d"]["guilds"]:
-            for channel in guild["channels"]:
-                if channel["id"] == self.channel_id:
-                    self.guild_id = guild["id"]
-                    break
+            try:
+                for channel in guild["channels"]:
+                    if channel["id"] == self.channel_id:
+                        self.guild_id = guild["id"]
+                        break
+            except:
```

Comment:
Black try except should not exist, raise required exceptions if needed or print out debug statements at the least.

Reply:
fixed